components/header/NavLinks.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { NavLink } from './NavLink';
+
+// Links will be hidden by default
+// But displayed on screens 768px and above
+const StyledNavLinks = styled.nav`
```

Comment:
As you said, you found one of the annoyances with styled-components: The naming. Here I'd simply name the component `Nav`. Alternatively I often use `Container` or `Wrapper` for the outer wrapper.

In general I'd refrain from using the `Styled` prefix as much as I can as it just adds clutter. So `Nav` or `Container` could be working here (with a tendency to `Nav` as it's more descriptive)

Another problem you'll run into is when you style imported components. E.g. if you'd want to add styles to the `NavLink` component you imported above. Some do this

```
import { NavLink } from "./NavLink";

const StyledNavLink = styled(NavLink)`...`
```

others something like this

```
import { NavLink as RawNavLink } from "./NavLink";

const NavLink = styled(RawNavLink)`...`
```